### ISSUE TITLE
What is going on with this library?

### ISSUE THREAD

**Issue by nosyke** [themesberg/flowbite-vue#188]
Recently started using Flowbite-Vue and I have to say, it's been a bit of a bumpy ride. First off, it's like finding a hidden gem (not in a good way) - the main site doesn't even point to it unless you happen to scroll all the way to the bottom where Flowbite advises actually you should use this other library we've halfway built because the main library doesn't actually work with Vue like you'd expect.

Then, when I finally got to it, it turns out its still not even working right. Ran into a bunch of issues that just seem like it needs a lot of work before this is ready for use. 

And don't get me started on the docs - what is even going on with those? 

Not trying to rant, but was really looking forward to using Flowbite and it's been a letdown so far. If I had known it was there were this many problems with Flowbite and Vue I would not have used it to begin with.

**Comment by gassio** [comment]
I totally agree with you. What I am trying to do is copy the component that I want to use in my project and make the necessary adjustments (functionality, styles).

I'm hoping that the Flowbite team will finish this project because I purchased their template and I feel like I bought something that doesn't meet my needs.

**Comment by zoltanszogyenyi** [comment]
Hey peeps,

I'm one of the maintainers of the core library - can you please give some specific feedback on what you'd like to see differently or improved with Flowbite Vue? It's still a work in progress so we are quite versatile with the API changes.

Cheers,
Zoltan

**Comment by gassio** [comment]
Hi @zoltanszogyenyi 
Thank you for your interest.
This is my personal opinion based on my needs, and I do not consider it to be better. 

These would be great if Flowbite-Vue had them:

1. Advanced Table functionality (filters, sorting, searching, etc) https://flowbite.com/blocks/application/advanced-tables/
2. Popover functionality (https://flowbite.com/docs/components/popover/)
3. Select with search functionality, Integration with vue-select (if possible) https://vue-select.org/
4. Add custom colors for Themable Configuration https://flowbite-vue.com/components/flowbiteThemable/flowbiteThemable

As I work on my project, I'll send you new ideas.

**Comment by gassio** [comment]
I'd like to add one more thing:
5. Drawer component implementation [https://flowbite.com/docs/components/drawer/](url)

Drawers are all too common these days. Many user interfaces use modals that are positioned on the side (left or right).

**Comment by gassio** [comment]
6. Datepicker functionality https://flowbite.com/docs/plugins/datepicker/
